hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -712,7 +717,17 @@ public void close() {
     }
 
     /**
-     * Clean up any stale/old files/data lying around (either on file storage or index storage)
+     * Clean up any stale/old files/data lying around (either on file storage or index storage) that is past
+     * the typical query timeout. Default is 12 hours.
+     */
+    public void clean() throws HoodieIOException {
+        String startCleanTime = startCommit();
+        clean(startCleanTime);
+    }
+
+    /**
+     * Clean up any stale/old files/data lying around (either on file storage or index storage) that is past
+     * the typical query timeout. Default is 12 hours.
```

Comment:
Same comment on index tagLocation and updateLocation...

Would come really handy, as we design the next generation of indexing,